code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm

```diff
@@ -196,7 +196,7 @@
 					choices += L
 		for(var/obj/O in oview(1,src))
 			if(Adjacent(O) && !O.anchored)
-				if(!istype(O, /obj/structure/spider/terrorweb) && !istype(O, /obj/structure/spider/cocoon) && !istype(O, /obj/structure/spider/spiderling/terror_spiderling))
+				if(!istype(O, /obj/structure/spider/terrorweb) && !istype(O, /obj/structure/spider/cocoon) && !istype(O, /obj/structure/spider/spiderling/terror_spiderling) && !istype(O, /obj/effect))
```

Comment:
So is that also why atmos can move bloodstains? Weird, I thought decals were by default anchored. At least the mess decals